examples/PaymentExample.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/**
+ * This file is part of the PHP Rebilly API package.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @see http://rebilly.com
+ */
+
+namespace Rebilly\Test;
+
+use Rebilly\v2_1\Payment;
+use Rebilly\v2_1\PaymentCardInstrument;
+use RebillyResponse;
+use Rebilly\HttpClient\HttpClient;
+use RebillyHttpStatusCode;
+
+/**
+ * Class PaymentExample.
+ */
+final class PaymentExample
+{
+    /** @var HttpClient */
+    private $client;
+
+    /**
+     * Create demo controller.
+     *
+     * @param HttpClient $client
+     */
+    public function __construct(HttpClient $client)
+    {
+        $this->client = $client;
+    }
+
+    /**
+     * Log message.
+     *
+     * @param string $message
+     */
+    private function log($message)
+    {
+        echo $message, PHP_EOL;
+    }
+
+    /**
+     * Demonstration how to list payments.
+     *
+     * @return RebillyResponse
+     */
+    public function listPayments()
+    {
+        $payment = new Payment();
+        $payment->setHttpClient($this->client);
```

Comment:
Why isn't the client set on the constructor if its going to be required to be set anyway?